lib/ace/mode/python_highlight_rules.js

```diff
@@ -165,10 +165,19 @@ var PythonHighlightRules = function() {
             regex: ",|:|;|\\->|\\+=|\\-=|\\*=|\\/=|\\/\\/=|%=|@=|&=|\\|=|^=|>>=|<<=|\\*\\*="
         }, {
             token: "paren.lparen",
-            regex: "[\\[\\(\\{]"
+            regex: "[\\(\\{]"
         }, {
             token: "paren.rparen",
-            regex: "[\\]\\)\\}]"
+            regex: "[\\)\\}]"
+        }, {
+            token: "paren.lbracket",
```

Comment:
Could you explain why yo want to a new token here? `paren.lbracket` seems to be some new token you introduced and it won't be styled properly anyway.

Reply:
You are right, I have removed these tokens from the pull request.